Makefile

```diff
@@ -1,34 +0,0 @@
-.PHONY: all clean deps fmt vet test docker
```

Comment:
Because the Makefile have been dropped for all 0.5 plugins. It's suggested to directly call `go build` and `go test`